### PR TITLE
feat(specs): add transformation copilot to ingestion

### PR DESCRIPTION
**Pull request by Fluf22**
## 🧭 What and Why

🎟 JIRA Ticket: -

### Changes included:

- add copilot handling in ingestion transformation specs

### CODE REVIEW

File: specs/ingestion/paths/transformations/transformationsCopilot.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+get:
+  tags:
+    - transformations
+  summary: Retrieve existing LLM transformation helpers
+  description: Retrieves a list of existing LLM transformation helpers.
+  operationId: listCopilots
```
Comment (shortcuts) [inline specs/ingestion/paths/transformations/transformationsCopilot.yml:6]: copilot is really close to github copilot, should we give it a more generic name?

File: specs/ingestion/common/schemas/transformation.yml [hunk 2]
```diff
@@ -119,3 +119,24 @@ TransformationTryResponse:
...
   required:
     - payloads
+
+Copilot:
+  type: object
+  additionalProperties: false
+  properties:
+    fullname:
+      type: string
+    modelName:
```
Comment (shortcuts) [inline specs/ingestion/common/schemas/transformation.yml:129]: should this be an enum, matching what we allow?

File: specs/ingestion/paths/transformations/transformationsCopilot.yml (hunk 1 above)
Comment (Fluf22) [reply specs/ingestion/paths/transformations/transformationsCopilot.yml:6]: Updated ✅

File: specs/ingestion/common/schemas/transformation.yml (hunk 2 above)
Comment (Fluf22) [reply specs/ingestion/common/schemas/transformation.yml:129]: I think this is premature, as we don't do any validation on the API side

File: specs/ingestion/paths/transformations/transformationsCopilot.yml (hunk 1 above)
Comment (shortcuts) [inline specs/ingestion/paths/transformations/transformationsCopilot.yml:6]: We need to update this too

File: specs/ingestion/paths/transformations/transformationsCopilot.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - addObject
+    - deleteIndex
+    - editSettings
+  responses:
+    '200':
+      description: OK
+      content:
+        application/json:
+          schema:
+            title: listTransformationModels
```
Comment (shortcuts) [inline specs/ingestion/paths/transformations/transformationsCopilot.yml:17]: I think this should be transformationModels, and the operationId should be this

File: specs/ingestion/paths/transformations/transformationsCopilot.yml (hunk 1 above)
Comment (Fluf22) [reply specs/ingestion/paths/transformations/transformationsCopilot.yml:6]: I'm stupid sorry

File: specs/ingestion/paths/transformations/transformationsCopilot.yml (hunk 3 above)
Comment (Fluf22) [reply specs/ingestion/paths/transformations/transformationsCopilot.yml:17]: Updated

File: specs/ingestion/spec.yml [hunk 4]
```diff
@@ -175,6 +175,8 @@ paths:
     $ref: 'paths/transformations/transformationsTry.yml'
   /1/transformations/search:
     $ref: 'paths/transformations/transformationsSearch.yml'
+  /1/transformations/copilot:
```
Comment (shortcuts) [inline specs/ingestion/spec.yml:178]: we need to change that too D: I'll add a ticket

**Review by shortcuts — APPROVED** [review]
looks great!

**Pull request merged**